packages/medusa/src/api/middlewares/transform-query.ts

```diff
@@ -12,12 +12,21 @@ import { FindConfig, QueryConfig, RequestQueryFields } from "../../types/common"
 import { omit } from "lodash"
```

Comment:
question: What happened in this file? 😅

Reply:
I have ported the update from develop to apply it in the handler 😅